tsdb/engine/tsm1/bool.go

```diff
@@ -0,0 +1,136 @@
+package tsm1
+
+// bool encoding uses 1 bit per value.  Each compressed byte slice contains a 1 byte header
+// indicating the compression type, followed by a variable byte encoded length indicating
+// how many booleans are packed in the slice.  The remaining bytes contains 1 byte for every
+// 8 boolean values encoded.
+
+import "encoding/binary"
+
+const (
+	// boolUncompressed is an uncompressed boolean format
+	boolUncompressed = 0
+	// boolCompressedBitPacked is an bit packed format using 1 bit per boolean
```

Comment:
Very minor, but why not just call it `boolUncompressed` and just leave it at that? Do you envision other compression formats going forward?